### ISSUE TITLE
Habitat does not display correctly for "Other" option

### ISSUE THREAD

**Issue by andrewvanbreda** [BiologicalRecordsCentre/PoMS#139]
If "Other" was selected by user then currently it displays "Other please describe below" which doesn't make sense to display on a grid. Changed to "Other" and precise detail in brackets.

Fixed for EU PoMS

Following reports altered:
	reports/projects/PoMS/blurred_poms_results_cache_4.xml
	reports/projects/PoMS/blurred_poms_results_lingual_4.xml
	reports/projects/PoMS/blurred_poms_results_mapping_lingual_4.xml
	reports/projects/PoMS/data_download_in_detail_with_blur_lingual_2.xml
	reports/projects/PoMS/my_poms_results_in_detail_cache_2.xml
	reports/projects/PoMS/my_poms_results_in_detail_lingual_4.xml
	reports/projects/PoMS/poms_results_in_detail_admin_cache_4.xml
	reports/projects/PoMS/poms_results_in_detail_admin_lingual_4.xml

Will await possible other changes before requesting pull

**Comment by kitenetter** [comment]
I don't think this fix has been applied to the UK Poms website, and it is a nice-to-have, but not a priority.

**Comment by andrewvanbreda** [comment]
@kitenetter 
I thought this had been fixed, but you are right it doesn't seem to be working at the moment. I will take a look some time when I get a second.

**Comment by andrewvanbreda** [comment]
Partially deployed (just the public Fit Count map page grid).
English and Welsh version of page changed.
Note that the word "Other" is currently displayed in the English on the Welsh page; I could use the Welsh google translation suggestion if you wish.
Welsh version of page was still incorrectly defaulting year to 2022, I have changed that.

**Comment by andrewvanbreda** [comment]
Hi @kitenetter 
Deployed to My Results and "List or Edit Samples" (for administrators). Both English and Welsh.

Old versions of reports removed.

Further problems fixed.
Welsh version of My Results was defaulting to 2022, I have removed that to match English version (no default year)

Welsh version of "List or Edit Samples" was defaulting to 2022, I have removed that to match English version 2025
